tests/integration/test_advanced_indexing.py

```diff
@@ -657,6 +672,34 @@ def test():
     x_num[ind_num, ind_num] = b_num
     assert np.array_equal(x, x_num)
 
+    # some additional tests for bool index arrays:
+    # 2d:
+    x = mk_seq_array(
+        np,
+        (
```

Comment:
formatting here is a bit silly. is this what the pre-commit hooks spit out?

Reply:
yes, it is pre-commit. I was trying to manually change it, but pre-commit will format again